### ISSUE TITLE
Feature request: $last support in ng-for

### ISSUE THREAD

**Issue by siggisim** [angular/angular#3102]
Would like to be able to do something like:

```html
<div *ng-for="var item of items">
      <div [class.selectable]="$last">Foo</div>
</div>
```

**Comment by matanlurey** [comment]
Just to clarify, you can do `var items of items, var last = items.length - 1` today, right?

**Comment by matanlurey** [comment]
Also, this makes sense for a `List`, but what if it is an `Iterable` and doing a length calculation is expensive?

**Comment by siggisim** [comment]
For an iterable, this would be true if hasNext is false. The length approach wouldn't work for iterables. A workaround that we're using right now is item == items.last, but this has issues if equals is overridden.

**Comment by pkozlowski-opensource** [comment]
This was already fixed by be954115f868d34e1d47626a3eebc0e56f78c0c1
